app/scripts/controllers/utils/utils.test.js

```diff
@@ -0,0 +1,92 @@
+import { strict as assert } from 'assert';
+import { checkSnapsBlockList } from './utils';
+
+describe('Controller utilities', function () {
+  describe('checkSnapsBlockList', function () {
+    it('returns one of the given snaps as blocked by its version', async function () {
+      const mockSnapsToBeChecked = {
+        'npm:exampleA': {
+          version: '1.0.0',
+          shasum: 'F5IapP6v1Bp7bl16NkCszfOhtVSZAm362X5zl7wgMhI=',
+        },
+        'npm:exampleB': {
+          version: '1.0.0',
+          shasum: 'eCYGZiYvZ3/uxkKI3npfl79kTQXS/5iD9ojsBS4A3rI=',
+        },
+        'npm:@consensys/starknet-snap': {
+          version: '0.1.10',
+          shasum: 'A83r5/ZIcKuKwuAnQHHByVFCuofj7jGK5hOStmHY6A0=',
+        },
+      };
+
+      const blockedSnaps = await checkSnapsBlockList(mockSnapsToBeChecked);
+      assert.deepEqual(blockedSnaps, {
```

Comment:
`assert` is node-specific while `expect` is jest specific. I haven't used that combination but error reporting might suffer due to lack of compatibility.

Reply:
I wanted and tried to use the `expect` but it seems that `assert`s are everywhere in the unit tests in this part of a project `/app/scripts/*`, etc. So, I'm not sure if I should introduce using `expect` here and if it will affect the test run 🤔. My editor also complains a bit if I do so: `ESLint: 'expect' is not defined.(no-undef)`, it seems like it's not configured, but it works. Controller tests use `assert` everywhere so I was following that: http://github.com/MetaMask/metamask-extension/blob/400009127bbd0c96400f60217a3b1f47035398f2/app/scripts/metamask-controller.test.js#L1230-L1230

Still, not sure what would be right in this case 🤔

Reply:
Following the standard practices in that repo should be good.